docs/overview/header.md

```diff
@@ -5,3 +5,74 @@ to the top of the viewport even when the main content area is scrolled up and do
 In order to create a header you have to create a CDI bean that implement org.uberfire.client.workbench.Header interface.
 
 ![header](header.png)
+
+Let us add a simple Composite that adds 3 labels. 
+
+In package org.uberfire.client.navbar we can create 
+
+- HeaderView.java
+
+```
+package org.uberfire.client.navbar;
+
+import com.google.gwt.user.client.ui.Composite;
+import com.google.gwt.user.client.ui.HorizontalPanel;
+import com.google.gwt.user.client.ui.Label;
+
+public class HeaderView extends Composite {
+
+    public HeaderView() {
```

Comment:
Also here it a view on the header, is there a need for a presenter ? the purpose is only to put the menu. Question : where and how can a add a menu to my view without using the default one and still using the setup method in ShowcaseEntrypoint ?